src/azul/indexer/__init__.py

```diff
@@ -286,7 +290,7 @@ def _read_contributions(self, tallies: Tallies) -> List[Contribution]:
         }
         index = sorted(list({config.es_index_name(e.entity_type, aggregate=False) for e in tallies.keys()}))
         # scan() uses a server-side cursor and is expensive. Only use it if the number of contributions is large
-        page_size = 100
+        page_size = 1000  # Using a large page size to reduce ScanError occurrences
```

Comment:
The problem with that comment is that *I* thought *100* is large. See comment in the line directly above.

```suggestion
        page_size = 1000  # page size of 100 caused excessive ScanError occurrences
```